home-application/src/main/java/com/softserveinc/ita/homeproject/application/config/SecurityConfig.java

```diff
@@ -42,7 +42,11 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .csrf().disable()
-                .httpBasic();
+                .httpBasic()
+                .and()
+                .authorizeRequests()
+                .anyRequest()
+                .authenticated();
```

Comment:
Please check:
 http://localhost:8090/actuator/health
 http://localhost:8080/api/0/apidoc.html

These ones should work without auth.

Reply:
Done